.github/workflows/bench-turbopack-scheduled.yml

```diff
@@ -0,0 +1,137 @@
+name: Benchmark Turbopack
+
+on:
+  schedule:
+    # Run every week on Sunday 12:42
+    - cron: "42 12 * * 0"
+  pull_request:
+    branches: [main]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+  NODE_OPTIONS: "--max-old-space-size=32768"
+
+jobs:
+  bench:
+    if: github.event_name == 'push' || startsWith(github.head_ref, 'bench/')
+    strategy:
+      fail-fast: false
+      matrix:
+        bench:
+          # One of Turbopack CSR or SSR with 1000 modules is benchmarked in every run
+          # to create a baseline result for normalization (should the runners performance vary between jobs)
+
+          # This measures Next.js 12
+          - name: next 12
+            cache_key: next-dev
+            args: -p next-dev -- "(Turbopack SSR/1000|Next.js 12)"
+            TURBOPACK_BENCH_COUNTS: 100,500,1000
+            TURBOPACK_BENCH_BUNDLERS: all
+
+          # This measures Next.js 11
+          - name: next 11
+            cache_key: next-dev
+            args: -p next-dev -- "(Turbopack SSR/1000|Next.js 11)"
+            TURBOPACK_BENCH_COUNTS: 100,500,1000
+            TURBOPACK_BENCH_BUNDLERS: all
+
+          # This measures Vite
+          - name: vite
+            cache_key: next-dev
+            args: -p next-dev -- "(Turbopack CSR/1000|Vite)"
+            TURBOPACK_BENCH_COUNTS: 100,500,1000
+            TURBOPACK_BENCH_BUNDLERS: all
+
+          # This measures Parcel
+          - name: parcel
+            cache_key: next-dev
+            args: -p next-dev -- "(Turbopack CSR/1000|Parcel)"
+            TURBOPACK_BENCH_COUNTS: 100,500,1000
+            TURBOPACK_BENCH_BUNDLERS: all
+
+          # This measures Webpack
+          - name: webpack
+            cache_key: next-dev
```

Comment:
`cache_key` doesn't need to be unique for each thing in the matrix?

Reply:
It depends only on the rust packages compiled: `-p next-dev` -> `cache_key: next-dev`